java/code/src/com/redhat/rhn/taskomatic/task/ForwardRegistrationTask.java

```diff
@@ -32,12 +32,17 @@
 
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.concurrent.ThreadLocalRandom;
 
 
 public class ForwardRegistrationTask extends RhnJavaJob {
 
+    // initialize first run between 30 minutes and 3 hours from now
+    private static LocalDateTime nextLastSeenUpdateRun = LocalDateTime.now().plusMinutes(
+            ThreadLocalRandom.current().nextInt(30, 3 * 60));
```

Comment:
What is the reason to wait before the first send? Multiple restarts of the server in a short time?

Reply:
yes, exactly